### PR TITLE
JENKINS-64069 Fix compat with core 2.264

### PR DESCRIPTION
**Pull request by timja**
cc @oleg-nenashev @fqueiruga @dwnusbaum 

for it to work fully it needs https://github.com/jenkinsci/cloudbees-folder-plugin/pull/165

Note: there's still other plugins (at least github-branch-source and workflow-multibranch) that need fixing that are causing JS errors but I couldn't see any actual problems.

### CODE REVIEW

**Review by res0nance — APPROVED** [review]

**Review by dwnusbaum — APPROVED** [review]
@timja Thanks! There aren't any problems for existing users if we release this in advance of https://github.com/jenkinsci/cloudbees-folder-plugin/pull/165, right? It's just that the UI will still be broken on 2.264 unless users have both fixes installed?

**Comment by timja** [comment]
Correct the UI will just get less broken.

There’s still other issues I haven’t tracked down yet but with a minimal plugin set this PR and the folders patch then I can create a scm source again

**Comment by timja** [comment]
@dwnusbaum  / @oleg-nenashev this is linked incorrectly in the release notes,

https://github.com/jenkinsci/branch-api-plugin/releases/tag/branch-api-2.6.1

clicking [JENKINS-64069] goes to https://issues.jenkins-ci.org/browse/JENKINS-59786

**Comment by fqueiruga** [comment]
From what I see we can close https://issues.jenkins-ci.org/browse/JENKINS-62825 right?

**Comment by timja** [comment]
> From what I see we can close [issues.jenkins-ci.org/browse/JENKINS-62825](https://issues.jenkins-ci.org/browse/JENKINS-62825) right?

yup done

**Comment by oleg-nenashev** [comment]
> clicking [JENKINS-64069] goes to https://issues.jenkins-ci.org/browse/JENKINS-59786

The link is fixed now

File: src/main/resources/lib/branch-api/rowWrapper.jelly [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:d="jelly:define" xmlns:st="jelly:stapler"> 
+  <st:documentation>
+    TODO remove and switch to div with tr class after baseline is 2.264 or newer
+    <st:attribute name="name"/>
+  </st:documentation>
+  <j:choose>
+    <j:when test="${divBasedFormLayout}">
+      <div class="tr" name="${attrs.name}">
```
Comment (jglick) [inline src/main/resources/lib/branch-api/rowWrapper.jelly:9]: [JENKINS-64069](https://issues.jenkins-ci.org/browse/JENKINS-64069)

File: src/main/resources/jenkins/branch/BranchSource/config.jelly [hunk 2]
```diff
@@ -23,18 +23,18 @@
  -->
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form" xmlns:branch="/lib/branch-api">
-  <tr name="source">
-    <td colspan="3">
+  <branch:rowWrapper name="source">
+    <branch:entryWrapper>
```
Comment (dwnusbaum) [inline src/main/resources/jenkins/branch/BranchSource/config.jelly:27]: Oops, this needs `colspan="3"`. Right now when running `branch-api` 2.6.1 against old versions of core it looks like this:

Screen Shot 2020-11-09 at 16 41 04

**Pull request merged**